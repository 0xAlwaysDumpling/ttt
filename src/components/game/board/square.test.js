import React from 'react'
import { render, cleanup } from '@testing-library/react'
import { Square } from './index';

afterEach(cleanup)

it('should render with out crashing', () => {
  render(<Square />)
});


it('should take a snapshot of square', () => {
  const { asFragment } = render(<Square />)

  expect(asFragment(<Square />)).toMatchSnapshot()
});
