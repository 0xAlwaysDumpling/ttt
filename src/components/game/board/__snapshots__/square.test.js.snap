// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxhCb exkQbL"
  >
    <button
      class="sc-AxjAm dRYkuE"
    />
  </div>
</DocumentFragment>
`;

exports[`should take a snapshot of square 1`] = `
<DocumentFragment>
  <div
    class="sc-AxhCb exkQbL"
  >
    <button
      class="sc-AxjAm dRYkuE"
    />
  </div>
</DocumentFragment>
`;
